src/safeds/data/tabular/containers/_time_series.py

```diff
@@ -292,19 +295,23 @@ def __hash__(self) -> int:
 
         Returns
         -------
-        hash:
+        hash : int
             The hash value.
         """
-        return _structural_hash(self.time, self.target, self.features, Table.__hash__(self))
+        return xxhash.xxh3_64(
+            hash(self.time).to_bytes(8)
+            + hash(self.target).to_bytes(8)
+            + hash(self.features).to_bytes(8)
+            + Table.__hash__(self).to_bytes(8),
+        ).intdigest()
```

Comment:
Please use the new hashing method implemented in #609

Reply:
this was on old main version